catroid/src/main/java/org/catrobat/catroid/utils/Utils.java

```diff
@@ -521,23 +521,32 @@ private static MessageDigest getMD5MessageDigest() {
 		ArrayList<String> lines = new ArrayList<>();
 
 		int cursorPos = 0;
-		while (cursorPos + Constants.MAX_STRING_LENGTH_BUBBLES < text.length()) {
-			String newLine = text.substring(cursorPos, cursorPos + Constants.MAX_STRING_LENGTH_BUBBLES);
-			int lastWhitespace = newLine.lastIndexOf(' ');
-			if (lastWhitespace < 0) {
-				lastWhitespace = Constants.MAX_STRING_LENGTH_BUBBLES;
+		int lineBreakIndex;
```

Comment:
since this function has only one usage in ShowBubbleActor, can you please move it there?